docs/game/unreal/v4.2.7.md

```diff
@@ -0,0 +1,283 @@
+* Download the Epic Games Launcher to access the Unreal Engine. [**Download**](https://www.unrealengine.com/en-US/download?sessionInvalidated=true)
+* Install Unreal Engine v5. [**Download**](https://www.unrealengine.com/en-US/download?sessionInvalidated=true)
+
+
+## STEP ONE - Add v4 Unreal Engine
+
+1. Open **Epic Games Launcher** and view all versions of the **Unreal Engine** you have. 
+
+2. Click **Library** and the **+** button to add the **v4** version
+
+    <img src={require('/img/unreal-v5.png').default} alt="blank" width="600" />
+
+3. You can now click to install **v4**. (This takes some time)
+
+4. Launch **v4** (Be patient, this can take a little while)
+
+   :::note
+
+   If a pop up appears:
+   "Do you want the app **UnrealEditor.app** to accept incoming network connections"
+   Select **Allow**
+
+   <img src={require('/img/game-mac-1.png').default} alt="blank" width="200" />
+
+   :::
+
+## STEP TWO - Create a new project
+
+1. Inside the **Unreal Project Browser**, select **Games** and click **Next**
+
+   <img src={require('/img/v4-games-2.png').default} alt="blank" width="500" />
+
+2. On the **Template** screen, select **Blank** and click **Next**
+
+    :::note
+
+    If you can't see any templates, you should reinstall v4
+
+    :::
+
+    <img src={require('/img/game-template.png').default} alt="blank" width="500" />
+
+3. In **Project Settings**
+* From the **Blueprint** and **C++** options, select **C++**. 
+* From the **Starter Content** and **No Starter Content** options, select **No Starter Content**
+* From the **Desktop/Console** or **Mobile** options, select your preference. 
+
+    <img src={require('/img/game-v4-2.png').default} alt="blank" width="500" />
+
+
+4. Give your project a memorable name e.g. BlockchainGame and then click **Create Project**
+
+5. Check your project is successfully created and opens. Then close it. So we can do the next important step of adding the SDK. 
+
+<img src={require('/img/game-mac-2.png').default} alt="create folder" width="350" />
+
+## STEP THREE - Download Ankr SDK
+
+1. Download the latest release of **AnkrSDK.zip** package from https://github.com/Ankr-network/game-unreal-sdk/releases.
+
+2. Create a new folder (if it doesn't exist) inside the project you just created and call it **Plugins**.
+
+3. Extract the downloaded **AnkrSDK.zip** contents into the **Plugins folder** you just created. 
+
+4. Delete the **AnkrSDK.zip** file if it is inside the **Plugins folder** as well. 
+
+<img src={require('/img/game-mac-3.png').default} alt="create folder" width="400" />
+
+
+## STEP FOUR - Generate the xcode project
+
+:::info 
+
+These docs are written using xcode 13.1
+
+:::
+
+1. Open **Unreal Editor**
+
+2. If you get a notice saying **"Missing BlockchainGame Modules Ankr SDK"**. Select **Yes** to rebuild.
+
+3. It should now compile the project. 
+
+4. Once the project is compiled, select **Tools** > **Generate xcode project**
```

Comment:
Does Apple really give its product a capital X at the beginning and then go and fuck up their menu like that — with a lowercase x?

Reply:
You are wrong. I am building this you are not.